### PR TITLE
Add setup instructions to README

### PR DESCRIPTION
**Pull request by thehenrybyrd**
I ran into some trouble building WordPressAuthenticator, and wanted to add instructions to make it more clear for other newcomers.
Please let me know if anything could be more clear here.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,6 +6,20 @@ This framework encapsulates everything required to display the Authentication UI
...
 Plus: WordPress.com *signup*  is supported.
 
+## Setup
+
+```sh
+# Install required ruby 2.6.5 
+rbenv install 2.6.5
+
+# Install bundler and Gemfile dependencies
+gem install bundle
```
Comment (ealeksandrov) [inline README.md]: typo 🙂
```suggestion
gem install bundler
```

File: README.md [hunk 2]
```diff
@@ -6,6 +6,20 @@ This framework encapsulates everything required to display the Authentication UI
 
 Plus: WordPress.com *signup*  is supported.
 
+## Setup
+
+```sh
+# Install required ruby 2.6.5 
+rbenv install 2.6.5
```
Comment (jkmassel) [inline README.md]: I'd suggest `rbenv install -s`

By not specifying the version, it's read from the `.ruby-version` file, so this instruction won't need to be updated if we update the project's ruby version.

The `-s` flag says "don't do anything if that version is already installed", which can be handy for folks who just copy/paste commands 😁

**Review by jkmassel — CHANGES_REQUESTED** [review]
I left one suggestion for improvement – thanks for doing this for everyone!

File: README.md (hunk 2 above)
Comment (thehenrybyrd) [reply README.md]: Oooh nice! Great suggestion, thank you!

**Review by ealeksandrov — APPROVED** [review]

**Review by jkmassel — APPROVED** [review]

**Pull request merged**